docs/proposal/dns_resolve.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Your short, descriptive title
+authors:
+- "@zhxuzhonghu"
+reviewers:
+- 
+approvers:
+- 
+
+
+creation-date: 2024-05-08
+
+---
+
+## Support DNS Resolution in Cluster Manager
+
+<!--
+This is the title of your KEP. Keep it short, simple, and descriptive. A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+
+### Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap.
+A good summary is probably at least a paragraph in length.
+-->
+
+Envoy supports many different cluster types, including `Strict Dns`, `Logical DNS`. However, given to kmesh works in the kernel with ebpf. Previously kmesh does not either of the DNS typed clusters. For traffic matches these kind of cluster, it will be dropped. 
```

Comment:
```suggestion
Envoy supports many different cluster types, including `Strict Dns`, `Logical DNS`. However, given to Kmesh works in the kernel with ebpf. Previously Kmesh does not either of the DNS typed clusters. For traffic matches these kind of cluster, it will be dropped. 
```